src/Module.ts

```diff
@@ -418,6 +447,25 @@ export default class Module {
 		}
 	}
 
+	getDynamicImportExpressions (): (string | Node)[] {
+		return this.dynamicImports.map(node => {
+			let dynamicImportExpression: string | Node;
```

Comment:
`dynamicImportExpression` becomes superfluous if you use early returns instead of the assignments.